.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: false  # Use container-based infrastructure
 language: python
 python:
   - "2.7"
-  - "3.4"
+  - "3.3"
```

Comment:
How come? I would have expected a bump to 3.6

Reply:
I was getting this error, any idea what random package uses this `enum` thing?

```
UnsatisfiableError: The following specifications were found to be in conflict:
  - enum34 -> python 3.3*
  - python 3.4*

Reply:
Now on 3.3 though I'm getting this one. Looks like asn1crypto just doesn't support Python 3. Not sure what package is pulling this in though

```
UnsatisfiableError: The following specifications were found to be in conflict:
  - asn1crypto -> python >=2.7,<2.8.0a0
  - python 3.3*